src/content/docs/style-guide/writing-docs/writer-workflow/github-intro.mdx

```diff
@@ -1,21 +1,23 @@
 ---
-title: Get around GitHub
+title: People, issues, and PRs
 topics:
   - Tech writer style guide
   - How to use GitHub
 redirects: 
   - /docs/style-guide/writer-workflow/github-intro
 ---
-As tech doc writers (TW) we [edit docs](/docs/style-guide/writer-workflow/tech-writer-workflow/), do [peer edits](/docs/style-guide/writer-workflow/peer-editor-workflow/), or use the [Docs Team GitHub board](https://github.com/newrelic/docs-website) to track the status of issues and pull requests (PR).  
+Tech writers (TW) are responsible for the docs. We [write and edit the docs](/docs/style-guide/writer-workflow/tech-writer-workflow/) and [exchange peer edits](/docs/style-guide/writer-workflow/peer-editor-workflow/), maintain our backlog, and `hero`. To make it all happen, we use our [Docs Team GitHub board](https://github.com/newrelic/docs-website) to track changes, file issues, and initiate pull requests (PRs).
```

Comment:
I like keeping "tech doc writers" even if it feels redundant with the "docs" later in the sentence.

Also, it isn't quite true that we use the GitHub board to file issues and initiate PRs. We specifically use it to track the status (or current state of work) of issues and PRs.

The TWers typically don't file issues (we don't need to, because we can just fix things) but we do create PRs.